packages/admin/cms-admin/src/blocks/createDamFileDownloadLinkBlock.tsx

```diff
@@ -0,0 +1,195 @@
+import { gql } from "@apollo/client";
+import { Field, FinalFormInput, FinalFormSelect } from "@comet/admin";
+import { Delete } from "@comet/admin-icons";
+import {
+    AdminComponentButton,
+    AdminComponentPaper,
+    AdminComponentSection,
+    BlockCategory,
+    BlockInterface,
+    BlocksFinalForm,
+    createBlockSkeleton,
+} from "@comet/blocks-admin";
+import { Box, Divider, MenuItem, Typography } from "@mui/material";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { DamFileDownloadLinkBlockData, DamFileDownloadLinkBlockInput } from "../blocks.generated";
+import { FileField } from "../form/file/FileField";
+import { CmsBlockContext } from "./CmsBlockContextProvider";
+import { GQLDamFileDownloadLinkFileQuery, GQLDamFileDownloadLinkFileQueryVariables } from "./createDamFileDownloadLinkBlock.generated";
+
+enum OpenFileTypeMethod {
+    NEW_TAB = "NEW_TAB",
+    DOWNLOAD = "DOWNLOAD",
+}
+type DamFileDownloadLinkBlockOptions = {
+    label?: React.ReactElement;
+    required?: boolean;
+};
```

Comment:
I believe this options were necessary in the project where this block is from, but they're probably not most of the time. Please remove the factory and required validation, and use a hardcoded value for the label